snap7/types.py

```diff
@@ -51,14 +52,24 @@
 mkEvent = 0
 mkLog = 1
 
+
 # Area ID
+class Areas(Enum):
```

Comment:
ADict could be use for typing as well right? If so, no need to use the enum

Reply:
Aparently ADict isn't as easy to use for typing as Enum, but switching everything to enum is cause some problems in other parts, it will take some time.